core/scss/components/coachmark/module.scss

```diff
@@ -1,15 +1,62 @@
-@import '../../tools/functions/core';
+@import '../../tools/mixins/core';
 
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import '../input/settings';
-@import '../select/settings';
-@import '../button/settings';
+// Settings
+@import '../../settings/settings';
 
+// Tools
+@import '../../tools/mixins/baseline';
 @import '../../tools/mixins/border-radius';
 @import '../../tools/mixins/flex';
-@import '../button/mixins';
+@import '../../tools/mixins/focus';
+@import '../../typography/mixins';
+@import '../../components/button/mixins';
+@import '../../components/checkbox/mixins';
+@import '../../components/input/mixins';
+@import '../../components/radio/mixins';
+@import '../../components/toggle-switch/mixins';
+
+@include brand-font-face-regular;
+@include brand-font-face-bold;
+
+// Generic
+@import '../../generic/global';
+
+// Components
+// extracted from @import '../../components/button/button';
+@include exports('md-button') {
```

Comment:
Is this for the button in the coachmark? or for the trigger? If it's the trigger, let's remove, since the trigger component should handle it's own styling.

Reply:
cool. removed